drivers/hmis/app/graphql/mutations/create_units.rb

```diff
@@ -13,14 +13,16 @@ class CreateUnits < BaseMutation
     def resolve(input:)
       project = Hmis::Hud::Project.viewable_by(current_user).find_by(id: input.project_id)
 
+      raise 'Not found' unless project.present?
+      raise 'Access denied' if project.present? && !current_user.permissions_for?(project, :can_manage_inventory)
```

Comment:
For a future iteration.  I wonder if these should be raising some sort of HmisError so that if we wanted to catch them we could, and they could be decorated with additional bits to allow for conditional catching...

Reply:
we raise `HmisErrors::ApiError` when there is additional context that we need to capture conditionally..but yeah maybe we should do that all the time